doc/adr/0001-use-micro-frontends.md

```diff
@@ -22,14 +22,15 @@ We investigated the use of micro frontends in [#371][Spike].
 
 ## Decision
 
-We will create separate services for individual, and groups of, pages, that will be exposed through a proxy to appear as
-a single service. The frontends are entirely independent, and can be not used or replaced entirely at will.
+We will create separate services for individual, and groups of, pages. The user will use a proxy to make them appear as
+a single service. The frontends are independent of each other, and are always optional.
```

Comment:
I see that this may mean not configuring a link transforms that link in simple text, so that the frontend it would point to becomes effectively optional.